README.md

```diff
@@ -45,6 +45,7 @@ Please, see [this guide](CONTRIBUTING.md), if you want to contribute to this lis
 - [IT KPI Rust](https://t.me/itkpi_rust) - **Telegram** - Чат ІТ KPI про Rust.
 - [IT KPI – Python](https://t.me/itkpi_python) - **Telegram** - Чат IT KPI обговорень про всякі штуки зі світу пайтону.
 - [PHP Friends Club Ukraine Chat](https://t.me/phpfriendsclub_chat) - **Telegram** - Чат про PHP та Web розробку.
+- [PyCON](https://www.meetup.com/uapycon/) - **Website** - Ukrainian Python User Groups
```

Comment:
```suggestion
- [PyCON](https://www.meetup.com/uapycon/) - **Website** - Ukrainian Python User Groups.
```

`awesome_lint` is unhappy if there is no dot.
Also, is this MeetUp group in Kyiv? If so, it probably makes sense to put it under `location.kyiv`